lib/src/features/symbol_line_renderer.dart

```diff
@@ -17,55 +14,80 @@ class SymbolLineRenderer extends FeatureRenderer {
   SymbolLineRenderer(this.logger);
 
   @override
-  void render(Context context, ThemeLayerType layerType, Style style,
-      TileLayer layer, TileFeature feature) {
+  void render(
+    FeatureRendererContext context,
+    ThemeLayerType layerType,
+    Style style,
+    TileLayer layer,
+    TileFeature feature,
+  ) {
     final textPaint = style.textPaint;
     final textLayout = style.textLayout;
     if (textPaint == null || textLayout == null) {
       logger.warn(() => 'line symbol does not have a text paint or layout');
       return;
     }
 
-    final lines = feature.lines;
-    logger.log(() => 'rendering linestring symbol');
+    // What if the feature has multiple paths?
+    final path = feature.paths.first;
```

Comment:
What's the plan here? Will we ever have multiple paths?

Thinking on how `LabelSpace` works, this might have the same effect if the first line ends up placing the label (since labels can only appear once in `LabelSpace`) but if the first path ends up with a collision with some other label, then another path might cause the label to be rendered.

I'm not sure if this ever worked before.

Reply:
My bad. I can just do this in the same the previous version did. 

Do you remember why you implemented `_findMiddleMetric` they way you did? Why not just iterate throuh all `PathMetric`s and use the first one that can be used. I could not find anything on the ordering of the result of `Path.computeMetrics`.

Reply:
But yes I found instances of symbol linestrings with multiple lines.

Reply:
Done